src/OrbitGl/App.cpp

```diff
@@ -1147,6 +1145,13 @@ ErrorMessageOr<std::filesystem::path> OrbitApp::FindSymbolsLocally(
   return ErrorMessage(error_message);
 }
 
+ErrorMessageOr<std::filesystem::path> OrbitApp::FindModuleLocally(
+    const std::filesystem::path& module_path, const std::string& build_id) {
+  const auto scoped_status = CreateScopedStatus(absl::StrFormat(
+      "Searching for symbols on local machine (module: \"%s\")...", module_path.string()));
```

Comment:
Especially since this ends up in the status bar, can we format it a bit better. The parentheses in particular make it a bit unusual. What about something like (but there's probably a better way and it depends on how other status bar message are formatted)
`"Searching for symbols on local machine for module: \"%s\"..."`
?